src/webgl/p5.Camera.js

```diff
@@ -709,14 +709,6 @@ p5.prototype.frustum = function (...args) {
 p5.prototype.createCamera = function () {
   this._assert3d('createCamera');
   const _cam = new p5.Camera(this._renderer);
-
-  // compute default camera settings, then set a default camera
-  _cam._computeCameraDefaultSettings();
-  _cam._setDefaultCamera();
```

Comment:
I think we still want these two lines, which set the camera settings to be a perspective camera with the defaultparameters. I believe it's just the line below that needs to be taken out to not have it set itself as the active camera.